src/webgpu/util/math.ts

```diff
@@ -880,6 +880,29 @@ export function biasedRange(a: number, b: number, num_steps: number): readonly n
   return Array.from(Array(num_steps).keys()).map(i => lerp(a, b, Math.pow(i / (num_steps - 1), c)));
 }
 
+/**
+ * Version of biasedRange that operates on bigint values
+ *
+ * biasedRange was not made into a generic or to take in (number|bigint),
+ * because that introduces a bunch of complexity overhead related to type
+ * differentiation
+ */
+export function biasedRangeBigInt(a: bigint, b: bigint, num_steps: number): readonly bigint[] {
```

Comment:
I know this is scuffed, the outputted range will be overly quantized, creating duplicates. I discovered this in a subsequent PR, I am working on a fix for this and will backport it to this PR.

Reply:
Fix backported